src/main/java/edu/harvard/iq/dataverse/DataverseConverter.java

```diff
@@ -26,7 +26,9 @@ public class DataverseConverter implements Converter {
 
     @Override
     public Object getAsObject(FacesContext facesContext, UIComponent component, String submittedValue) {
-        return dataverseService.find(new Long(submittedValue));
+        var pk = (submittedValue == null || submittedValue.isEmpty() || submittedValue.matches(".*[^0-9].*"))
+        ? 0 : Long.valueOf(submittedValue);
```

Comment:
FWIW - the root dataverse does not always have id=0

Reply:
FWIW: There is a findRoot call that could be used instead.

Reply:
@jo-pol can you please switch this to `dataverseService.findRootDataverse()` instead? Thanks.